services/nuget/nuget-v2-service-family.js

```diff
@@ -20,26 +19,11 @@ function createFilter({ packageName, includePrereleases }) {
   const releaseTypeFilter = includePrereleases
     ? 'IsAbsoluteLatestVersion eq true'
     : 'IsLatestVersion eq true'
-  return `Id eq '${packageName}' and ${releaseTypeFilter}`
+  return `tolower(Id) eq '${packageName.toLowerCase()}' and ${releaseTypeFilter}`
```

Comment:
Case now seems to matter on the Chocolatey API. Without this, our tests based on `scriptcs` fail, we'd need to use `ScriptCs`. I'm guessing other users may have relied on case-insensitivity in the past, so this keeps backwards compatibility.